config/300-cephsource.yaml

```diff
@@ -18,6 +18,7 @@ metadata:
   labels:
     ceph.eventing.knative.dev/release: devel
     eventing.knative.dev/source: "true"
+    duck.knative.dev/source: "true"
```

Comment:
this is good.

Reply:
https://github.com/knative-sandbox/discovery/blob/main/config/knative/sources.duck.knative.dev.yaml#L23